main.go

```diff
@@ -19,16 +20,17 @@ func main() {
 		return
 	}
 
-	var logger *zap.Logger
+	var logger *slog.Logger
 	switch {
```

Comment:
we're creating different setups for development and production here. Could you explain more about how this logging distinction should work? This will help us customize the slog process more effectively.

Reply:
`debug` level logs shouldn't be logged in production